bot/cogs/moderation/infractions.py

```diff
@@ -220,7 +220,7 @@ def __init__(self, bot: Bot):
     @respect_role_hierarchy()
     async def apply_kick(self, ctx: Context, user: Member, reason: str, **kwargs) -> None:
         """Apply a kick infraction with kwargs passed to `post_infraction`."""
-        infraction = await utils.post_infraction(ctx, user, "kick", reason, **kwargs)
+        infraction = await utils.post_infraction(ctx, user, "kick", reason, active=False, **kwargs)
```

Comment:
Just curious, was  this a previously undetected bug? (We typically don't kick people anymore.)

Reply:
Do you mean that `active` would be `True` for a kick? Yes, it was an undetected bug. As I wrote in the other issue, the validator in the serializer could be by bypassed by not including the value at all.

Reply:
In the command route, the `active=False` is already provided by the command, so it's passed on as a kwarg:
```py
    @command()
    async def kick(self, ctx: Context, user: Member, *, reason: str = None) -> None:
        """Kick a user for the given reason."""
        await self.apply_kick(ctx, user, reason, active=False)
```

Reply:
Ah well it may as well stay in `apply_kick` too.